examples/webpack.config.local.js

```diff
@@ -53,7 +55,8 @@ function makeLocalDevConfig(EXAMPLE_DIR = LIB_DIR, linkToLuma) {
     devServer: {
       stats: {
         warnings: false
-      }
+      },
+      contentBase: [ROOT_DIR, resolve(ROOT_DIR, '..' , 'loaders.gl') ]
```

Comment:
Don't check this in

Reply:
It's only for debugging purpose, will be removed when draft is removed